src/sentry/tasks/integrations/github/pr_comment.py

```diff
@@ -1,8 +1,44 @@
+from dataclasses import dataclass
+from typing import List
+
 from django.db import connection
 
 from sentry.models import GroupOwnerType
 
 
+@dataclass
+class PullRequestIssue:
+    title: str
+    subtitle: str
+    url: str
+
+
+body = """## Suspect Issues
+This pull request has been deployed and Sentry has observed the following issues
+
+{issue_list}
+
+<sub>Did you find this useful? React with a \U0001F44D or \U0001F44E</sub>"""
+
+single_issue_template = "- ‼️ **{title}** `{subtitle}` [View Issue]({url})"
```

Comment:
Can you use the Unicode for ‼️ too?

Reply:
putting the emojis back because the escaped unicode doesn't get recognized by the JSON parser :(